neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -229,6 +254,12 @@ void IVerifiable.SerializeUnsigned(BinaryWriter writer)
             writer.Write(Attributes);
             writer.Write(Inputs);
             writer.Write(Outputs);
+
+            if (Version >= 2)
+            {
+                writer.Write(CreationHeight);
+                writer.Write(Nonce);
```

Comment:
User can select any CreationHeight? Even if it's very old? It could pre-compute all tx before, to prepare mass attacks. I don't think Nonce is necessary, if Remark fields are already being used to provide transaction randomization.